rokka/sourceimages.go

```diff
@@ -122,3 +126,36 @@ func (c *Client) CreateSourceImageWithMetadata(org, name string, data io.Reader,
 
 	return result, err
 }
+
+func (c *Client) DynamicMetadataResponseHandler(v interface{}, resp *http.Response) error {
```

Comment:
this doesn't need to be a function receiver of `Client` (i.e. can be a standalone, private function)